napari_allencell_annotator/_tests/widgets/file_item_test.py

```diff
@@ -8,15 +8,31 @@
 from napari_allencell_annotator.widgets.file_item import FileItem, QLabel, QCheckBox
 
 
+def test_get_name():
+    # ACT
+    tiff_file_name: str = FileItem(Path("parent/test.tiff"), QListWidget(), True).get_name()
+    png_file_name: str = FileItem(Path("parent/test.png"), QListWidget(), True).get_name()
+    jpeg_file_name: str = FileItem(Path("parent/test.jpeg"), QListWidget(), True).get_name()
+    czi_file_name: str = FileItem(Path("parent/test.czi"), QListWidget(), True).get_name()
+    zarr_file_name: str = FileItem(Path("parent/test.zarr"), QListWidget(), True).get_name()
+
+    # ASSERT
+    assert tiff_file_name == "test"
+    assert png_file_name == "test"
+    assert jpeg_file_name == "test"
+    assert czi_file_name == "test"
+    assert zarr_file_name == "parent"
+
```

Comment:
again only really need to test one image and one zarr

Reply:
also shouldnt this really be in fileutils? and for tests, parent should be `None` instead of a random QListWidget so it uses less memory.

Reply:
I ended up removing the get_name function